l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol

```diff
@@ -34,7 +36,7 @@ contract MailboxFacet is ZkSyncHyperchainBase, IMailbox {
     string public constant override getName = "MailboxFacet";
 
     /// @dev Era's chainID
-    uint256 immutable ERA_CHAIN_ID;
+    uint256 public immutable ERA_CHAIN_ID;
```

Comment:
It shouldn't be public. This is diamond proxy facet, if you make the function public then it will be a imported as a getter for proxy that is unexpected to be a part of Mailbox and not Getters

Reply:
ill add it to the changes for https://github.com/matter-labs/era-contracts/pull/477